### DISCUSSION TITLE
How can I get teacher's benifits

### DISCUSSION THREAD

**Discussion by jyluo1994** [community/community#40639] [GitHub Education]
Hi, I processed my application for teacher's benifits it returned the following:
"Your application cannot be reviewed until you fix the following:
Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?,If you meant to apply as a student, please cancel your application and reapply [here](https://education.github.com/discount_requests/student_application). ,You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio."
But it seems that I have completed my information. What should I do?

**Comment by samus-aran** [comment]
Hi @jyluo1994 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

Hope that helps 👍
